pilot/pkg/config/kube/gateway/testdata/http.yaml

```diff
@@ -274,3 +274,28 @@ spec:
     timeouts:
       request: 2ms
       backendRequest: 1ms
+---
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: http-retry-request
+  namespace: default
+spec:
+  parentRefs:
+    - name: gateway
+      namespace: istio-system
+  hostnames: ["retry.domain.example"]
+  rules:
+    - matches:
+        - path:
+            type: PathPrefix
+            value: /get
+      backendRefs:
+        - name: httpbin
+          port: 80
+      retry:
```

Comment:
Can you please add a test case that changes the attempts?

Reply:
:+1: added a few more (and caught an issue!)